test/cmdlineargs.jl

```diff
@@ -257,6 +257,16 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
         @test expanded == readchomp(addenv(`$exename -e 'println(Base.active_project())'`, "JULIA_PROJECT" => "@foo", "HOME" => homedir()))
     end
 
+    # --project=@script handling
+    let expanded = abspath(joinpath(@__DIR__, "project", "ScriptProject"))
+        script = joinpath(expanded, "bin", "script.jl")
+        # Check running julia with --project=@script both within and outside the script directory
+        @testset "--@script from $name" for (name, dir) in [("project", expanded, ("outside", pwd()))]
```

Comment:
It looks like this `for` loop has a misplaced closing parenthesis:
```julia
[("project", expanded, ("outside", pwd()))]
                                         ^
```
I'm guessing should be
```julia
[("project", expanded), ("outside", pwd())]
                     ^
```
Otherwise this loop has only one iteration and ignores `"outside"` and `pwd()`.

Reply:
Good catch, thank you!